### ISSUE TITLE
Custom model

### ISSUE THREAD

**Issue by MinhQuang2021** [clovaai/spade#8]
I found Layoutlmv2 takes both the image and the context into the pretrain. Why would you use BERT and not Layoutlmv2. I want custom to use layoutlmv2 for encoder, is it possible? Thank you so much

**Comment by whwang299** [comment]
Hi @MinhQuang2021 

You may customize this part 
https://github.com/clovaai/spade/blob/a85574ceaa00f1878a23754f283aa66bc2daf082/spade/model/model.py#L743-L775
The backbone is based on `hugginface transformers` thus it shouldn't be difficult to adapt `layoutlm`. 

During the major development , `layoutlm` wasn't available. Also `layoutlm` is not pre-trained for Japanese and Indonesian. Thus we initialized the `spade encoder` with the subset of BERT weights (note that `spade encoder` ≠ `BERT`).

Best

**Comment by MinhQuang2021** [comment]
Thank you, I really want to apply it on my custom dataset for my language. But creating a dataset with a label like CORD is quite time consuming, so I want to ask you, How many datasets will give good results? Is the standard dataset for this model important? Is this model applicable to unseen data well?
Can you give me some knowledge in EL task I still find it quite confusing in paper to just generate graph and decode it.
I am learning it for my graduation project. Thank you very much

**Comment by MinhQuang2021** [comment]
![image](https://user-images.githubusercontent.com/91367856/143218147-5faeb375-2f46-456e-85a3-7350bd2c9471.png) 
I noticed when set self.hparam.token_lv_boxing == True. parse is parsed differently. Why in Cord_test.yml and Cord_train.yml you both set it to True. Can you explain how it works for me?

**Comment by MinhQuang2021** [comment]
why do you set token_lv_boxing= True in CORD , and in FUNSD you set token_lv_boxing=False .
How does token_lv_boxing work? I noticed it changes the parse result when training.

**Comment by MinhQuang2021** [comment]
hello, can you help me find the code to apply inferring_method?
     - tca_rel_s
     - tca_rel_g

**Comment by whwang299** [comment]
Hi @MinhQuang2021 
I would say, roughly 1,000 documents are required for the reasonable performance. 
Under zero-shot setting, the domain should be similar for reasonable performance. Otherwise, you'll observe a large performance drop.

About the EL task, FUNSD consists of four types of fields: "Header" "Question", "Answer", "ETC". 
The EL task aims to connect "Header" → "Question", "Question" → "Answer".
So it is kind of "grouping two fields where each field consist of multiple serialized words". 

When `token_lv_boxing=True`, `spade` draws directed arrows between tokens. For the noisy OCR data like receipt, this makes a better result. For FUNSD, the task assumes a perfect OCR so I turned it off.

The data augmentation option is automatically turned off during prediction. Sorry for the confusion.

Sorry for being late in reply. Good luck with your project!
